app/src/main/java/com/github/kpdandroid/ui/DetectionResultColumn.kt

```diff
@@ -0,0 +1,56 @@
+package com.github.kpdandroid.ui
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun DetectionResultColumn(
+    imageLayers: List<ImageBitmap>,
```

Comment:
I use `Pair` in `ImageAnalysisViewModel` because it interracts with exactly two layers and distinguishes them based on their position in the pair: this logic, if replaced with a collection, will either be broken, or become unsafe. On the contrary, composables can work with arbitrary amount of layers and don't need to know what each layer represents.